addons/pandora/ui/editor/inspector/entity_instance_inspector.gd

```diff
@@ -1,6 +1,11 @@
 extends EditorInspectorPlugin
 
 const BrowserProperty = preload("res://addons/pandora/ui/editor/inspector/entity_instance_browser_property.gd")
+const PANDORA_ENTITY_CLASS = &"PandoraEntity"
+
+
+# ClassName -> Dictionary
+var _global_class_cache = {}
```

Comment:
We have to maintain a class cache so we can lookup class dependency structures. Only exporting custom types of type `PandoraEntity` should be eligible for selection.